src/ostree/ot-builtin-summary.c

```diff
@@ -55,10 +109,16 @@ ostree_builtin_summary (int argc, char **argv, GCancellable *cancellable, GError
 
   if (opt_update)
     {
+      g_autoptr(GVariant) additional_metadata = NULL;
+
       if (!ostree_ensure_repo_writable (repo, error))
         goto out;
 
-      if (!ostree_repo_regenerate_summary (repo, NULL, cancellable, error))
+      additional_metadata = argv_to_additional_metadata (&argc, &argv, error);
```

Comment:
Yeah, I'd vote for making it into long & short opts, if at least to be consistent with `ostree commit`'s `--add-metadata-string`. Compare to e.g. `ostree ls`, where the user more intuitively knows what the extra args do.

/bikeshed :)